examples/demo/app/globals.css

```
@@ -124,5 +124,59 @@
 	}
 	body {
 		@apply bg-background text-foreground;
+		min-height: 100vh;
```

Stylelint (17.7.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)